src/Combinatorics/SimplicialComplexes.jl

```diff
@@ -280,6 +286,38 @@ Multivariate Polynomial Ring in 6 variables a, b, c, d, ..., f over Integer Ring
 """
 stanley_reisner_ring(R::FmpzMPolyRing, K::SimplicialComplex) = quo(R, stanley_reisner_ideal(R, K))
 
+function fundamental_group(K::SimplicialComplex)
+    n, r = pm_object(K).FUNDAMENTAL_GROUP
+    F = free_group(n)
+    return fundamental_group(F, K)
+end
+
+@doc Markdown.doc"""
+    fundamental_group([F::FPGroup,] K::SimplicialComplex)
+
+Return the fundamental group of the abstract simplicial complex `K`, as a quotient of a given Group `F`.
```

Comment:
Group should be lower case...
```suggestion
Return the fundamental group of the abstract simplicial complex `K`, as a quotient of a given group `F`.
```
But more importantly:
- how is `F` chosen if not given?
- what does `F` have to satisfy to be suitable? Should it perhaps be a free group of a certain rank? If it not a free group, and/or has a different rank, what is the result?!?